docs/INSTALL.md

```diff
@@ -12,9 +12,11 @@
 
 ## Deploy the Operator
 
-Deploy the operator by running the following command.
+Deploy the operator by running the following command where `<RELEASE_VERSION>` needs to be substituted
+with the desired [release tag](https://github.com/confidential-containers/operato/tags).
+
 ```
-kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+kubectl apply -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
```

Comment:
Users will want to just copy and paste the instructions.
Would be possible to have <RELEASE_VERSION> as something automagically populated with the latest release?

Reply:
I take this back, this is not that simple to do without assuming users have some specific tools installed on their OSes.